internal/services/managedredis/managed_redis_data_source_test.go

```diff
@@ -51,6 +52,21 @@ func TestAccManagedRedisDataSource_basic(t *testing.T) {
 	})
 }
 
+func TestAccManagedRedisDataSource_dbPersistence(t *testing.T) {
```

Comment:
Data sources don't typically have more than 1 test (complete), can we just extend the existing test to accommodate the new properties?